github/GitRelease.py

```diff
@@ -187,9 +187,11 @@ def update_release(
         prerelease: bool = False,
         tag_name: Opt[str] = NotSet,
         target_commitish: Opt[str] = NotSet,
+        make_latest: Opt[str] = NotSet,
+        discussion_category_name: Opt[str] = NotSet,
     ) -> GitRelease:
         """
-        :calls: `PATCH /repos/{owner}/{repo}/releases/{release_id} <https://docs.github.com/en/rest/reference/repos#update-a-release>`_
+        :calls: `PATCH /repos/{owner}/{repo}/releases/{release_id} <https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#update-a-release>`_
         """
         assert tag_name is NotSet or isinstance(tag_name, str), "tag_name must be a str/unicode object"
         assert target_commitish is NotSet or isinstance(
```

Comment:
Please add the assertion here, remove the next to lines, and move all these four assertions below `assert isinstance(prerelease, bool), prerelease`, to reflect  the order of arguments in the method signature:

```suggestion
        assert is_optional(tag_name, str), "tag_name must be a str/unicode object"
        assert is_optional(target_commitish, str), "target_commitish must be a str/unicode object"
        assert make_latest in ["true", "false", "legacy", NotSet], make_latest
        assert is_optional(discussion_category_name, str), discussion_category_name
```

Reply:
done. I also resorted the `post_parameters` statements to have the same order. Only the defaulting of `tag_name` is out of order but I think it more unified now, thanks for the heads-up!